### ISSUE TITLE
Get password from environment var and encrypt an unencrypted file

### ISSUE THREAD

**Issue by iangregsondev** [sadv1r/ansible-vault-editor-idea-plugin#52]
Great plugin!

Would be nice if you could get the password from the standard environment variable so not asking it each time

Also would be nice to be able to encrypt a non encrypted file,.

As far as I can see, you can unencrypt and re encrypt and already encrypted by not when the encryption hasn't been run once..

**Comment by sadv1r** [comment]
@iangregsondev Thanks, for participating! New file yaml property encryption is really needed and will be added soon.

What about your other suggestion. Currently there is a way to store passwords in [default IDEA key store](https://www.jetbrains.com/help/idea/reference-ide-settings-password-safe.html). You only need to activate checkbox "Remember password" while entering the password.
Do you have use cases where you really need the password in environment variable instead?

**Comment by iangregsondev** [comment]
@sadv1r Thanks for the response. With regards to remembering password, this seems to popup on every single (new) file I try to access.

Some people do store their password in an ENV variable so it would be really handy to work like that.

I came from using a plugin in vs code and it had this functionality which was nice - but it had other issues which is why I moved over to using the IDEA pllugin.
